ci-operator/config/openshift/hive/openshift-hive-master.yaml

```diff
@@ -125,28 +125,20 @@ tests:
           cpu: 100m
     workflow: generic-claim
 - as: e2e-pool
-  cluster_claim:
-    architecture: amd64
-    as: unused
-    cloud: aws
-    labels:
-      region: us-east-1
-    owner: openshift-ci
-    product: ocp
-    timeout: 1h0m0s
-    version: "4.12"
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
+    cluster_profile: aws-2
     test:
     - as: test
       cli: latest
       commands: |
         # Creds for the `ci` user in the hive-team cluster
-        # TODO: Get a CI-owned account: DPTP-2348
-        export AWS_ACCESS_KEY_ID=$(cat /tmp/secret/hive-aws-creds/AWS_ACCESS_KEY_ID)
-        export AWS_SECRET_ACCESS_KEY=$(cat /tmp/secret/hive-aws-creds/AWS_SECRET_ACCESS_KEY)
-        # Override default location for pull secret
-        export PULL_SECRET_FILE=/tmp/secret/pull-secret/.dockerconfigjson
+        export AWS_ACCESS_KEY_ID=$(cat /tmp/hivesecret/hive-aws-creds/AWS_ACCESS_KEY_ID)
+        export AWS_SECRET_ACCESS_KEY=$(cat /tmp/hivesecret/hive-aws-creds/AWS_SECRET_ACCESS_KEY)
```

Comment:
See [DPTP-3490](https://issues.redhat.com/browse/DPTP-3490) for context on `/tmp/secret` => `/tmp/hivesecret`.